evalai/submissions.py

```diff
@@ -114,32 +127,44 @@ def push(image, phase):
     federated_user = response["success"]["federated_user"]
     repository_uri = response["success"]["docker_repository_uri"]
 
-    AWS_ACCOUNT_ID = federated_user["FederatedUser"]["FederatedUserId"].split(
-        ":"
-    )[0]
-    AWS_SERVER_PUBLIC_KEY = federated_user["Credentials"]["AccessKeyId"]
-    AWS_SERVER_SECRET_KEY = federated_user["Credentials"]["SecretAccessKey"]
-    SESSION_TOKEN = federated_user["Credentials"]["SessionToken"]
+    # Default Environment
+    if ENVIRONMENT == "PRODUCTION":
```

Comment:
This `if` statement will come up after line 122, no?

Reply:
Please ignore this, I think if we will merge this one, then we won't need this change: https://github.com/Cloud-CV/EvalAI/pull/2359/files